File: javascript/explore.js
let TWILIO_AUTH_SID;
let TWILO_AUTH_TOKEN;
const Twilio = require("twilio");

const client = new Twilio(
  "AC2a7c23dc33b299ebd312a453c9adb7b8",
  "bc4ff5f356d4a11a1f18c88d0801b78d"
);

client.messages
  .list()
  .then((messages) =>
    console.log(`The most recent message is ${messages[0].body}`)
  ).catch((err) =>console.error(err));

console.log("Gathering your messages.....");

File: python/explore.py
from twilio.rest import Client

# Your Account SID from twilio.com/console
account_sid = "Your Account SID"

# Your Auth Token from twilio.com/console
auth_token  = "Your Auth Token"

client = Client(account_sid, auth_token)

# Check messages sent
''' for msg in client.messages.list():
    print(msg.body) '''


# Create and send message
#msg = client.messages.create(to="number", from_="number",body="The message you want to send",)

# To deleted your messages 
''' for msg in client.messages.list():
    print(f"Deleting {msg.body}")
    msg.delete()   '''